launch/tier4_localization_launch/launch/localization.launch.xml

```diff
@@ -6,10 +6,13 @@
   <arg name="random_downsample_filter_param_path"/>
   <arg name="ndt_scan_matcher_param_path"/>
   <arg name="localization_error_monitor_param_path"/>
+  <arg name="pose_initializer_param_path"/>
+  <arg name="pose_initializer_param_logging_simulator_path"/>
```

Comment:
Why are you passing two config file paths? It is unnecessary if you specify the config file path according to the system run mode.

Reply:
Thanks. There's no particular reason. Resolved: https://github.com/autowarefoundation/autoware.universe/pull/3038/commits/99b590e8aca2fb7dd109e94c26949e0f1d2bde88